manifest.php

```diff
@@ -40,7 +41,7 @@
         [
             AccessRule::GRANT,
             'http://www.tao.lu/Ontologies/TAO.rdf#PropertyManagerRole',
-            ['controller' => 'oat\taoBackOffice\controller\Lists'],
+            ['act' => Lists::class .'@getListElements'],
```

Comment:
Lets also check if the previous bug fixed for UDIR wont come back after this change. This change will be deployed to them, so we might run this migrations in the dev environment for UDIR and test it.

Reply:
I've tested this in dev & testing envs for UDIR and the call to getListElements is working fine both for a admin and author users.